components/abstractions/gradle/dependencies.gradle

```
@@ -7,10 +7,12 @@ dependencies {
     // Use JUnit Jupiter Engine for testing.
     testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine'
 
+    //This dependency is used internally and also by Kiota generated code which leverages this library.
+    api 'jakarta.annotation:jakarta.annotation-api:2.1.1'
```

```suggestion
    implementation 'jakarta.annotation:jakarta.annotation-api:2.1.1'
```
I thought we had agreed to leave as implementation and add the dependency in the app settings? Cc @andreaTP

---

Ah yes, I reverted the changes and forgot to push.